code/modules/core_implant/cruciform/rituals/crusader.dm

```diff
@@ -0,0 +1,63 @@
+/datum/ritual/cruciform/crusader
+	name = "crusader"
+	phrase = null
+	desc = ""
+	category = "Crusader"
+
+/datum/ritual/cruciform/crusader/battle_call
+	name = "Call to Battle"
+	phrase = "Deus, qui corda fidolium Sancti Spiritus illustratione docuisti."
+	desc = "Inspires the prayer and gives him strength to protect the other cyberchristians. True strength in unity."
+	cooldown = TRUE
+	cooldown_time = 10 MINUTES
+	cooldown_category = "battle call"
+	effect_time = 10 MINUTES
+
+/datum/ritual/cruciform/crusader/battle_call/pre_check(mob/living/carbon/human/H, obj/item/weapon/implant/core_implant/C, targets)
+	if(cooldown && is_on_cooldown(H))
+		fail("This type of litany can't be spoken too often.", H, C)
+		return FALSE
+	return TRUE
+
+/datum/ritual/cruciform/crusader/battle_call/perform(mob/living/carbon/human/user, obj/item/weapon/implant/core_implant/C)
+	var/count = 0
+	for(var/mob/living/carbon/human/brother in view(user))
+		if(brother.get_cruciform())
+			count++
+
+	user.stats.changeStat(STAT_TGH, count)
```

Comment:
this is a pretty small buff, i'd say do count * 4

Reply:
That will snowball out of control. Remember that there are a lot of litanies that boost stats, so it's normal to have small numbers on each one.

Reply:
I think *2 should be more or less optimal.